### PR TITLE
Initial packaging

### PR DESCRIPTION
**Pull request by dseight**

### CODE REVIEW

File: rpm/firejail.spec [hunk 1]
```diff
@@ -0,0 +1,38 @@
+Name: firejail
+Version: 0.9.62
+Release: 1
+Summary: Linux namepaces sandbox program
+License: GPLv2+
+Group: Development/Tools
```
Comment (rainemak) [inline rpm/firejail.spec]: Should not be used: https://sailfishos.org/wiki/Software_Packaging#RPM_packaging_Guidelines

File: rpm/firejail.spec [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+%configure \
+    --disable-x11 \
+    --disable-overlayfs \
+    --disable-contrib-install
+%make_build
+
+%install
+%make_install
+rm -rf %{buildroot}%{_datadir}/bash-completion
+rm -rf %{buildroot}%{_datadir}/doc
```
Comment (rainemak) [inline rpm/firejail.spec]: Part of doc package

File: rpm/firejail.spec [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+rm -rf %{buildroot}%{_mandir}
+
+%files
+%doc
+%defattr(-,root,root,-)
+%attr(4755, -, -) %{_bindir}/firejail
```
Comment (rainemak) [inline rpm/firejail.spec]: %license COPYING

File: rpm/firejail.spec [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+%make_build
+
+%install
+%make_install
+rm -rf %{buildroot}%{_datadir}/bash-completion
+rm -rf %{buildroot}%{_datadir}/doc
+rm -rf %{buildroot}%{_mandir}
+
+%files
+%doc
```
Comment (rainemak) [inline rpm/firejail.spec]: %license COPYING

**Comment by rainemak** [comment]
Minor remarks. Generally looking good.

**Comment by saukko** [comment]
Commit message missing bug reference.

**Comment by rainemak** [comment]
Amend commit message... to "[firejail] Initial packaging. Contributes JB#49256"

File: rpm/firejail.spec (hunk 1 above)
Comment (dseight) [reply rpm/firejail.spec]: `Group` removed

File: rpm/firejail.spec (hunk 2 above)
Comment (dseight) [reply rpm/firejail.spec]: Moved docs to `-doc` package

File: rpm/firejail.spec (hunk 4 above)
Comment (dseight) [reply rpm/firejail.spec]: Done

**Pull request merged**